test/general/key.js

```diff
@@ -3934,4 +4028,48 @@ XvmoLueOOShu01X/kaylMqaT8w==
     expect(signingKeySignature instanceof openpgp.SignaturePacket).to.be.true;
     expect(signingKeySignature.keyFlags[0] & openpgp.enums.keyFlags.signData).to.be.equals(openpgp.enums.keyFlags.signData);
   });
+
+  it("Sign and verify key - key with user attribute", async function () {
+    let publicKey = await openpgp.readKey({ armoredKey: keyWithImageData });
+    const privateKey = await openpgp.decryptKey({
+      privateKey: await openpgp.readKey({ armoredKey: priv_key_rsa }),
+      passphrase: "hello world",
```

Comment:
The linter complains that this should be
```suggestion
      passphrase: 'hello world'
```
(and also the name of the test should use single quotes :blush:)

Reply:
updated, thank you.